tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "dist",
+    "strict": true,
+    "target": "es6",
+    "module": "commonjs",
```

Comment:
I would use esnext as module

Reply:
Update on the tsconfig: Project building is no longer necessary, because of the Bun runtime. This means, that these configurations can be omitted completely. The project uses ESNext modules and not commonjs FYI